rubocop.yml

```diff
@@ -1027,3 +1027,7 @@ Lint/DeprecatedOpenSSLConstant:
 
 Lint/Syntax:
   Enabled: true
+
+Style/WordArray:
+  EnforcedStyle: brackets
+  Enabled: true
```

Comment:
You don't need this line. By declaring the cop with a config it is going to be enabled already.